app/views/api/v3/analytics/user_analytics/_diagnosis_report.html.erb

```diff
@@ -32,12 +32,6 @@
     <p class="m-0px mb-24px p-0px ta-left fw-normal fs-16px lh-150 c-grey-dark">
       <%= t("progress_tab.diagnosis_report.assigned_patients_card.subtitle", facility_name: current_facility.name, diagnosis: report_diagnosis.downcase) %>
     </p>
-    <% diagnosis_data["assigned_patients"]["table_breakdown"].each_with_index do |row, index| %>
-      <%= render partial: "api/v3/analytics/user_analytics/data_table_plusminus_row",
-                 locals: { title: row["title"], value: row["value"],
-                           css_text_color: row["value"] > 0 ? "c-green-new" : "c-red-new",
-                           include_bottom_border: index == diagnosis_data["assigned_patients"]["breakdown"].length - 1 ? false : true } %>
-    <% end %>
```

Comment:
What's this removal?

Reply:
Removed assigned patients breakdown .ie `Registered` , `Transferred-in`, `Transferred-out` from the Hypertension report